contracts/clients/README.md

```diff
@@ -77,14 +96,161 @@ const bundle = wallet.sign({
 await aggregator.add(bundle);
 ```
 
+### Sending a regular ETH transaction
+
+```ts
+// Follow the same steps as the first BlsWalletWrapper example, but construct the bundle actions like so:
+const amountToTransfer = ethers.utils.parseUnits("1");
+const reciever = "0x1234...";
+
+const bundle = wallet.sign({
+  nonce,
+  actions: [
+    {
+      ethValue: amountToTransfer, // amount of ETH you want to transfer
+      contractAddress: reciever, // receiver address. Can be a contract address or an EOA
+      encodedFunction: "0x", // leave this as "0x" when just sending ETH
+    },
+  ],
+});
+```
+
+### Constructing actions to be agnostic to both ETH transfers and contract interactions
+
+```ts
+// Follow the same steps as the first BlsWalletWrapper example, but construct the bundle actions like so:
+const tranactions = [
+  {
+    value: ethers.utils.parseUnits("1"), // amount of ETH you want to transfer
+    to: "0x1234...", // to address. Can be a contract address or an EOA
+  },
+];
+
+const actions: ActionData[] = tranactions.map((tx) => ({
+  ethValue: tx.value ?? "0",
+  contractAddress: tx.to,
+  encodedFunction: tx.data ?? "0x", // in this example, there is no data property on the tx object, so "0x" will be used
+}));
+
+const bundle = wallet.sign({
+  nonce,
+  actions,
+});
+```
+
+## Estimating and paying fees
```

Comment:
@jacque006 This section covers a chunk of the feedback from the hackmd doc you shared. Let me know if this covers fees well. I changed the examples slightly but let me know if you think they need additional changes.

Also more generally, if you feel like I've missed anything from your hackmd doc, feel free to add yourself or let me know and I'll make the changes.